.taskcluster.yml

```diff
@@ -5,385 +5,278 @@
-              in:
-                $mergeDeep:
-                  - {$eval: 'default_task_definition'}
-                  - {$eval: 'release_task_definition'}
-                  - scopes:
-                    - assume:repo:${event.repository.html_url[8:]}:release
-                  - payload:
-                      env:
-                        IS_STAGING: ${is_staging}
-                        BEETMOVER_WORKER_TYPE: ${beetmover_worker_type}
-                        BEETMOVER_BUCKET: ${beetmover_bucket}
-                        BEETMOVER_BUCKET_PUBLIC_URL: ${beetmover_bucket_public_url}
-                  - metadata:
-                      name: Application Services - Decision task (${tag})
-                      description: Build and publish release versions.
+                              maxRunTime: 1800
+
+                              command:
+                                  - /usr/local/bin/run-task
+                                  - '--appservices-checkout=/builds/worker/checkouts/src'
+                                  - '--taskgraph-checkout=/builds/worker/checkouts/taskgraph'
+                                  - '--task-cwd=/builds/worker/checkouts/src'
+                                  - '--'
+                                  - bash
+                                  - -cx
+                                  - >
+                                    PIP_IGNORE_INSTALLED=0 pip install --user /builds/worker/checkouts/taskgraph &&
+                                    PIP_IGNORE_INSTALLED=0 pip install --user arrow taskcluster pyyaml &&
+                                    ln -s /builds/worker/artifacts artifacts &&
+                                    ~/.local/bin/taskgraph decision
+                                    --pushlog-id='0'
+                                    --pushdate='0'
+                                    --project='${project}'
+                                    --message=""
+                                    --owner='${ownerEmail}'
+                                    --level='${level}'
+                                    --base-repository="$APPSERVICES_BASE_REPOSITORY"
+                                    --head-repository="$APPSERVICES_HEAD_REPOSITORY"
+                                    --head-ref="$APPSERVICES_HEAD_REF"
+                                    --head-rev="$APPSERVICES_HEAD_REV"
+                                    --head-tag="$APPSERVICES_HEAD_TAG"
+                                    --repository-type="$APPSERVICES_REPOSITORY_TYPE"
+                                    --tasks-for='${tasks_for}'
+                              artifacts:
+                                  'public':
+                                      type: 'directory'
+                                      path: '/builds/worker/artifacts'
+                                      expires: {$fromNow: '1 year'}
+                          extra:
+                            $merge:
+                                - treeherder:
```

Comment:
We don't (yet) have TH support for application-services so likely below is useless, for now. However, I'm keeping it here for the near future when hopefully we can migrate this project there too.

Reply:
👍 We don't use `tc-treeherder.*` routes so taskcluster won't try to feed treeherder with these values. It's fine to keep them here, in my opinion.